packages/react-instantsearch-hooks-dom/src/ui/InfiniteHits.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+
+import type { Hit } from 'instantsearch.js';
+import type { JSXElementConstructor } from 'react';
+
+export type InfiniteHitsProps<THit> = React.ComponentProps<'div'> & {
+  hitComponent?: JSXElementConstructor<{ hit: THit }>;
+  hits: THit[];
+  isFirstPage: boolean;
+  isLastPage: boolean;
+  showPrevious?: () => void;
+  showMore: () => void;
```

Comment:
This is more consistent with other UI components taking callbacks (`onNavigate`, etc.)

```suggestion
  onShowPrevious?: () => void;
  onShowMore: () => void;
```